storage_controller/src/service/safekeeper_service.rs

```diff
@@ -502,6 +506,17 @@ impl Service {
                 sk.1.id.0,
             )
         });
+        // Number of safekeepers in different AZs we are looking for
+        let wanted_count = match all_safekeepers.len() {
+            0 => {
+                return Err(ApiError::InternalServerError(anyhow::anyhow!(
+                    "couldn't find any active safekeeper for new timeline",
+                )));
+            }
+            1 => 1,
+            2 => 2,
```

Comment:
If for some reason we only have 2 sks in a production region, then shouldn't we fail the request?